engine/table/src/test/java/io/deephaven/engine/table/impl/by/TestSortedFirstOrLastByFactory.java

```diff
@@ -54,11 +51,14 @@ public void testSortedFirstOrLastBy() {
     private void incrementalTest(int seed, int size, final String... sortColumns) {
         final Random random = new Random(seed);
         final ColumnInfo<?, ?>[] columnInfo;
+
+        final String[] colNames = new String[] {"Sym", "intCol", "doubleCol", "Indices"};
+
         final QueryTable queryTable = getTable(size, random, columnInfo = initColumnInfos(
                 colNames,
-                new SetGenerator<>("a", "b", "c", "d"),
+                new SetGenerator<>("a", "b", "c", "d", "e", "f"),
                 new IntGenerator(10, 100, 0.1),
-                new SetGenerator<>(10.1, 20.1, 30.1),
+                new SetGenerator<>(10.1, 20.1, 30.1, NULL_DOUBLE, Double.NaN),
```

Comment:
I think we should be worried that we are going to end up with too many things poisoned given that 20% of rows will be NaN.

Perhaps add some tests that remove the NaN or have a separate column that excludes the NaN.